src/odemis/acq/test/fastem_test.py

```
@@ -135,10 +135,10 @@ def test_overview_acquisition(self):
         before_start_t = time.time()
         f = fastem.acquireTiledArea(self.stream, self.stage, scintillator5_area)
         time.sleep(1)
-        start_t, end_t = f.get_progress()
-        self.assertGreater(start_t, before_start_t)
+        elapsed_t, remaining_t = f.get_progress()
```

Ruff (0.15.21)

[warning] Unpacked variable `remaining_t` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)